05_stringer/brvishnu92/main_test.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+//main() test
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+	main()
+	expected := `127.0.0.1
+`
+	actual := buf.String()
+	if expected != actual {
```

Comment:
Have a look at [testify/assert](https://godoc.org/github.com/stretchr/testify/assert), you've used it in your second test!

Reply:
Sure, changed it.. :-)